index.js

```diff
@@ -1,8 +1,8 @@
 const fs = require('node:fs');
-const { Client, Collection, Intents } = require('discord.js');
+const { Client, Collection } = require('discord.js');
 const { token, countingCh } = require('./config.json');
 
-const client = new Client({ ws: { properties: { $browser: "Discord iOS" }}, intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_VOICE_STATES, Intents.FLAGS.GUILD_MESSAGES, Intents.FLAGS.GUILD_MESSAGE_REACTIONS] });
```

Comment:
If you want to know, you can use a integer here, to calculate the integer i used [This app](https://discord-intents-calculator.vercel.app/)

Reply:
which means that you have more ram available when you run your program!